reproducibility_project/src/engines/mcccs/project.py

```diff
@@ -109,14 +187,27 @@ def has_topmon(job):
 
 @Project.label
 def has_fort77maker(job):
-    """Check if the job has a fort77maker file (obsolete)."""
+    """Check if the home dir has fort77maker for one box and two box simulations."""
     return os.path.isfile(
         Project().root_directory()
         + "/src/engines/mcccs/"
         + "fort77maker_onebox.py"
+    ) and os.path.isfile(
+        Project().root_directory()
+        + "/src/engines/mcccs/"
+        + "fort77maker_twobox.py"
     )
 
 
+@Project.label
+def equil_replicate_set(job):
+    """Check if number of equil replicates done has been set."""
+    try:
+        return isinstance(job.doc.equil_replicates_done, int)
+    except AttributeError:
+        return False
```

Comment:
```suggestion
    return isinstance(job.doc.get("equil_replicates_done"), int)
```
I think you can simplify this logic like this. The `get` function will return `None` if the key does not exist, which will return `False` from `isinstance(None, int)`